packages/esm-patient-search-app/src/patient-search-context.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+interface PatientSearchContextProps {
+  nonNavigationSelectPatientAction?: (patientUuid: string) => void;
+  patientClickSideEffect?: ((patientUuid: string) => void) | (() => void);
```

Comment:
It would probably be good to include some doc strings on how these are intended to be used? I can't really think of clearer names per-se, but I'm not sure they're really self-describing...